packages/cozy-intent/src/api/services/NativeService.ts

```diff
@@ -73,21 +61,25 @@ export class NativeService {
           : messenger
       }
     }
+
+    log(strings.logging.registered(uri))
   }
 
-  public unregisterWebview = (webviewRef: WebviewRef): void => {
-    const uri = this.getUri(webviewRef)
+  public unregisterWebview = (uri: string): void => {
```

Comment:
new API

Reply:
that is breaking change

Reply:
I notified this in the PR. Won't this trigger the semver? I thought the PR description was enough to trigger a semver major update

Reply:
Semantic release uses every commits, especially each commit description to detect breaking change.

The content of PR description is not added inside the git history, that's why setting breaking change in PR won't provoke a breaking change.

Reply:
updated